docs/docs/color-filters.md

```diff
@@ -10,33 +10,34 @@ slug: /color-filters
 Creates a color filter using the provided color matrix.
 A playground to build color matrices is available [here](https://fecolormatrix.com/). 
 
-| Name       | Type          |  Description                               |
-|:-----------|:--------------|:-------------------------------------------|
-| value      | `number[]`    | Color Matrix (5x4)                         |
-| children?  | `ColorFilter` | Optional color filter to be applied first. |
+| Name      | Type          |  Description                               |
+|:----------|:--------------|:-------------------------------------------|
+| matrix    | `number[]`    | Color Matrix (5x4)                         |
+| children? | `ColorFilter` | Optional color filter to be applied first. |
 
 ```tsx twoslash
-import { Canvas, Paint, ColorMatrix, Image } from "@shopify/react-native-skia";
+import { Canvas, Paint, ColorMatrix, Image, useImage } from "@shopify/react-native-skia";
 
 const MatrixColorFilter = () => {
+  const image = useImage(require("./assets/oslo.jpg"));
   return (
     <Canvas style={{ flex: 1 }}>
       <Paint>
         <ColorMatrix
-          value={[
+          matrix={[
             -0.578, 0.99, 0.588, 0, 0, 0.469, 0.535, -0.003, 0, 0, 0.015,
             1.69, -0.703, 0, 0, 0, 0, 0, 1, 0,
           ]}
         />
       </Paint>
-      <Image
+      { image && (<Image
```

Comment:
If image has falsy values like `"", 0`, it might crash with 
```
[Invariant Violation: Text strings must be rendered within a <Text> component](https://stackoverflow.com/questions/52368342/invariant-violation-text-strings-must-be-rendered-within-a-text-component)
```

It's probably better to use conditional operator here or wrap the condition `image` with `Boolean` instead.